src/wp-content/themes/twentynineteen/style-editor.css

```
@@ -605,7 +605,7 @@ body .wp-block.aligncenter {
 }
 
 @media only screen and (min-width: 768px) {
-  body.block-editor-writing-flow,
+  body.block-editor-iframe__body, body.block-editor-writing-flow,
```

I'm being picky, but it would be nice to see each selector on its own line.
```suggestion
  body.block-editor-iframe__body,
  body.block-editor-writing-flow,
```

---

Yes, it would be nicer on its own line, but the stylesheet compiled that way. I do not know how to force a line break there.